lib/ecto_state_machine.ex

```diff
@@ -29,6 +29,20 @@ defmodule EctoStateMachine do
         unquote(events) |> Enum.map(fn(x) -> x[:name] end)
       end
 
+      def unquote(:"get_event_to")(event) do
+        unquote(events)
+        |> Enum.find(
```

Comment:
This doesn't work well for me. What could be if there are couple of ways to go from point A to point B?